### PR TITLE
add: getCustomErrorMessage() and tests, bump version, rename to just helpers

### PR DESCRIPTION
**Pull request by mikemaccana**
- add getCustomeErrorMessage(), docs, and tests
 - rename to just 'helpers' https://imgflip.com/i/8bedom
 - bump version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,77 @@
...
 
 ## Installation
 
 ```bash
 npm i @solana-developers/node-helpers
 ```
 
+# helpers for the browser and node.js
+
+## getCustomeErrorMessage()
```
Comment (Woody4618) [inline README.md:15]: this should probably be getCustomErrorMessage

File: src/index.ts [hunk 2]
```diff
@@ -11,6 +11,23 @@ export const keypairToSecretKeyJSON = (keypair: Keypair): string => {
...
+export const getCustomErrorMessage = (
+  possibleProgramErrors: Array<string>,
+  errorMessage: string,
+): string | null => {
+  const customErrorExpression =
+    /.*custom program error: 0x(?<errorNumber>[0-9abcdef]+)/;
+
+  let match = customErrorExpression.exec(errorMessage);
+  const errorNumberFound = match?.groups?.errorNumber;
+  if (!errorNumberFound) {
```
Comment (Woody4618) [inline src/index.ts:23]: Probably makes more sense to return the error instead of null

**Review by Woody4618 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mikemaccana) [reply README.md]: Good spot, fixed.

File: src/index.ts (hunk 2 above)
Comment (mikemaccana) [reply src/index.ts:23]: The function is called `getCustomErrorMessage()` - if there's no custom error message, it should return nothing, as we could not find a custom error message. Returning the original error would be deceptive.

**Pull request merged**